document/core/text/types.rst

```diff
@@ -139,7 +152,8 @@ Memory Types
 .. math::
    \begin{array}{llclll@{\qquad\qquad}l}
    \production{memory type} & \Tmemtype &::=&
-     \X{lim}{:}\Tlimits &\Rightarrow& \X{lim} \\
+     \X{lim}{:}\Tlimits &\Rightarrow& \I32~\X{lim} \\ &&|&
```

Comment:
This should be defined separately now, like other abbreviations.

Reply:
Done